src/main/webapp/app/view/project/screenplay/project-screenplay.component.ts

```diff
@@ -0,0 +1,82 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute, Params, Router } from '@angular/router';
+import { HttpResponse } from '@angular/common/http';
+
+import { faCog } from '@fortawesome/free-solid-svg-icons';
+
+import { Location } from 'app/shared/model/location.model';
+import { Project } from 'app/shared/model/project.model';
+
+import { LocationService } from 'app/entities/location/location.service';
+import { NgbPanelChangeEvent } from '@ng-bootstrap/ng-bootstrap';
+
+const ROUTE_PARAM_NAME = 'activeLocations';
+
+@Component({
+  selector: 'app-screenplay',
+  templateUrl: './project-screenplay.component.html',
+  styleUrls: ['project-screenplay.component.scss'],
+})
+export class ProjectScreenplayComponent implements OnInit {
+  faCog = faCog;
+
+  project?: Project;
+  locations?: Location[];
+
+  activeLocations: string[] = [];
+
+  constructor(private locationService: LocationService, private router: Router, private activatedRoute: ActivatedRoute) {}
+
+  ngOnInit(): void {
+    this.activatedRoute.data.subscribe(({ project }) => {
+      this.project = project;
+
+      this.locationService
+        .findAllByProject(project, {
+          unpaged: true,
```

Comment:
Dachte die Methode wurde in einem PR angepasst, dass es gar keine Optionen mehr gibt, da wir das Pageination nicht mehr benötigen

Reply:
das ist korrekt, die ist etwas überflüssig..